steampipeconfig/load_mod_test.go

```diff
@@ -408,3 +409,67 @@ func executeLoadTest(t *testing.T, name string, test loadModTest, wd string) {
 		t.Errorf("Test: '%s'' FAILED : expected:\n\n%s\n\ngot:\n\n%s", name, expectedStr, actualString)
 	}
 }
+
+type loadResourceNamesTest struct {
+	source   string
+	expected interface{}
+}
+
+var testCasesLoadResourceNames map[string]loadResourceNamesTest
+
+func init() {
```

Comment:
you can actually just define the tests cases in the global var, the same as the workspace test I linked - the mod load test is a special case

Reply:
Made the change in e16f0936849f0f5531d2ad7b686d66daf2026cd3